### ISSUE TITLE
Is there a way to invoke tensorboard for cross encoder model training? 

### ISSUE THREAD

**Issue by mucun1988** [UKPLab/sentence-transformers#2430]
I am wondering whether there is a way to invoke tensorboard for visualization when training cross encoder model? 
E.g., https://github.com/UKPLab/sentence-transformers/blob/master/examples/training/cross-encoder/training_nli.py#L73

**Comment by tomaarsen** [comment]
Hello!

I'm afraid that at this time it is not easily possible to get the loss in Tensorboard or W&B during training. The only option currently is to use the [`callback` option](https://sbert.net/docs/package_reference/cross_encoder.html#sentence_transformers.cross_encoder.CrossEncoder.fit) and create a function that logs the scores to Tensorboard. However, this is likely more difficult than imagined.

I'm actively working on implementing integrations with third-parties like Tensorboard to make this much, much easier in the future.

- Tom Aarsen

**Comment by l4b4r4b4b4** [comment]
same question here. Did not see it, so guess only way it tto implement this yourself on the callback function.

**Comment by tomaarsen** [comment]
Indeed, that's currently the only way. I'm planning on improving the training over the CrossEncoder much like I've done for the SentenceTransformer. By then, the tensorboard integration will be automatic.

- Tom Aarsen

**Comment by l4b4r4b4b4** [comment]
sounds great. Can you give a time estimate on when you think youll release it? 

Will you include Hyper-param optimisation like with sentence transformer?
